tox.ini

```diff
@@ -126,8 +126,7 @@ deps =
     pep8-naming==0.11.1
     pycodestyle==2.6.0
     pydocstyle==5.1.1
-    # pin pyflakes pending a release with https://github.com/PyCQA/pyflakes/pull/455
-    git+git://github.com/PyCQA/pyflakes@ffe9386#egg=pyflakes
+    pyflakes==2.2.0
```

Comment:
This change was necessary because of a [version conflict caught by Travis](https://travis-ci.org/github/python-hyper/hyperlink/jobs/751084720#L234):

```
flake8 create: /home/travis/build/python-hyper/hyperlink/.tox/flake8

flake8 installdeps: flake8-bugbear==20.11.1, flake8==3.8.4, mccabe==0.6.1, pep8-naming==0.11.1, pycodestyle==2.6.0, pydocstyle==5.1.1, git+git://github.com/PyCQA/pyflakes@ffe9386#egg=pyflakes

ERROR: invocation failed (exit code 1), logfile: /home/travis/build/python-hyper/hyperlink/.tox/flake8/log/flake8-1.log

================================== log start ===================================

Collecting pyflakes

  Cloning git://github.com/PyCQA/pyflakes (to revision ffe9386) to /tmp/pip-install-_gaosggb/pyflakes_9797c9a25e194e17ae54355c8ab0ac64

  WARNING: Did not find branch or tag 'ffe9386', assuming revision or ref.

Collecting flake8==3.8.4

  Using cached flake8-3.8.4-py2.py3-none-any.whl (72 kB)

ERROR: Cannot install flake8==3.8.4 and pyflakes 2.1.1 (from git+git://github.com/PyCQA/pyflakes@ffe9386#egg=pyflakes) because these package versions have conflicting dependencies.

The conflict is caused by:

    The user requested pyflakes 2.1.1 (from git+git://github.com/PyCQA/pyflakes@ffe9386#egg=pyflakes)

    flake8 3.8.4 depends on pyflakes<2.3.0 and >=2.2.0

To fix this you could try to:

1. loosen the range of package versions you've specified

2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```